app/src/ui/rebase/rebase-branch-dialog.tsx

```diff
@@ -160,6 +160,8 @@ export class RebaseBranchDialog extends React.Component<
       return
     }
 
+    // TODO: transition to a rebase progress dialog
```

Comment:
Is there a PR for this piece or is this todo for a later date?

Reply:
This isn't found in a PR currently, but #6962 is the related issue and this requires reworking the flow between components